### PR TITLE
Find annotation function

### PR DESCRIPTION
**Pull request by lyriccoder**
Fix https://github.com/cqfn/aibolit/issues/500
It's a part of https://github.com/cqfn/aibolit/issues/462

### CODE REVIEW

File: aibolit/__main__.py [hunk 1]
```diff
@@ -162,6 +163,32 @@ def flatten(l):
     return [item for sublist in l for item in sublist]
 
 
+def find_annotation_by_node_type(tree, node_type):
```
Comment (acheshkov) [inline aibolit/__main__.py:166]: please, add typing to the function signature

File: aibolit/__main__.py [hunk 2]
```diff
@@ -162,6 +163,32 @@ def flatten(l):
...
+                elif hasattr(a.element, 'values'):
+                    for j in a.element.values:
+                        if 'aibolit' in j.value:
+                            annonations[node].append(
+                                j.value.split('.')[1].rstrip('\"')
+                            )
+    return annonations
+
+
 def find_start_and_end_lines(node):
```
Comment (acheshkov) [inline aibolit/__main__.py:192]: please, add typing to the function signature

File: test/recommend/test_recommend_pipeline.py [hunk 3]
```diff
@@ -255,3 +255,35 @@ def test_find_start_end_line_in_class(self):
             start, end = find_start_and_end_lines(method)
             self.assertEqual(start, 11)
             self.assertEqual(end, 59)
+
+    def test_find_annotation_by_class_declaration(self):
+        file = Path(self.cur_file_dir, 'annotations/ClassAnnotations.java')
+        with open(file, 'r', encoding='utf-8') as f:
+            tree = javalang.parse.parse(f.read())
+            classes_with_annonations = find_annotation_by_node_type(tree, javalang.tree.ClassDeclaration)
+            pattern_found = list(classes_with_annonations.values())[0][0]
```
Comment (acheshkov) [inline test/recommend/test_recommend_pipeline.py:264]: can we check against all found annotations?

Comment (lyriccoder) [reply test/recommend/test_recommend_pipeline.py:264]: this test is about class declaration only

File: aibolit/__main__.py (hunk 2 above)
Comment (lyriccoder) [reply aibolit/__main__.py:192]: This is dict, any type can be inside

**Comment by acheshkov** [comment]
@rultor merge

**Pull request merged**